app/views/support_interface/settings/cycles.html.erb

```diff
@@ -38,7 +38,7 @@
   'Apply 2 deadline' => CycleTimetable.apply_2_deadline.to_s(:govuk_date),
   'Find closes on' => CycleTimetable.find_closes.to_s(:govuk_date),
   'Find reopens on' => CycleTimetable.find_reopens.to_s(:govuk_date),
-  'Apply reopens on' => CycleTimetable.apply_reopens.to_s(:govuk_date),
+  'Apply reopens on' => CycleTimetable.apply_opens.to_s(:govuk_date),
```

Comment:
This should be `.reopens`

Reply:
Yes!
https://github.com/DFE-Digital/apply-for-teacher-training/pull/5106/commits/14ecdf560c1cb5b42c301c526ef381748cf26e9a